conjure-undertow-lib/src/main/java/com/palantir/conjure/java/undertow/lib/Parameters.java

```diff
@@ -0,0 +1,81 @@
+/*
```

Comment:
I think this belongs in `lib.internal` since it's not meant to be used by code written by humans

Reply:
Sounds good!